subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GrettySmokeTest.groovy

```diff
@@ -60,7 +61,8 @@ class GrettySmokeTest extends AbstractPluginValidatingSmokeTest {
 
         when:
         def result = runner('checkContainerUp')
-            .expectDeprecationWarning(
+            .expectDeprecationWarningIf(
```

Comment:
This warning isn't present anymore, it should be a hard error instead if it happens in any of the tests.

Reply:
That makes sense, we'll never go back to testing an earlier version.